theme/static/js/resourceLandingAjax.js

```diff
@@ -936,8 +936,8 @@ function remove_aggregation_ajax_submit(url) {
         success: function (result) {
             waitDialog.dialog("close");
             $("#fb-inner-controls").before($alert_success);
-            $(".alert-success").fadeTo(2000, 500).slideUp(1000, function(){
-                $(".alert-success").alert('close');
+            $("#success-alert").fadeTo(2000, 500).slideUp(1000, function () {
+                $("#success-alert").alert('close');
```

Comment:
I'm easing back into vanilla JS/jQuery after a stint last year in ReactJS/ES6... is this function supposed to have the paren with a whitespace? `function ()`

Reply:
The whitespace makes no difference functionality-wise, but it makes the code style more appealing. It is also what is recommended in standard formatting by IDEs like PyCharm.

Reply:
Noted. I was just going by the GH interface, obviously. I ran the code via command line and didn't look at the linting.